composer.json

```diff
@@ -33,7 +33,7 @@
 		}
 	],
 	"require": {
-		"php": ">5.3.0,~5.5.0"
+		"php": ">5.3.0,~5.6.0"
```

Comment:
I don't see a reason for this change, you're not using any 5.6 specific features, are you? The other thing is to try to keep this in line with the TYPO3 LTS requirements.